src/protocols/near/schema.ts

```diff
@@ -38,6 +38,7 @@ type MethodDefinition = {
 }
 
 export async function getContractData(contract?: string): Promise<SerializableContractData | undefined> {
+  debugger;
```

Comment:
❓

Reply:
This is before I understood it was serverside.

It's all a mess.